Author: Jan Christoph Ebersbach <jceb@e-jc.de>
URL: http://dwm.suckless.org/patches/historical/raiselower
This patch enables you to raise or lower the selected floating window.

--- dwm.orig//dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ dwm/raiselower.c	Sat Feb 11 21:13:22 2012 +0100
@@ -0,0 +1,20 @@
+void
+raisewindow (const Arg *arg) {
+	if (!selmon->sel)
+		return;
+	if(selmon->lt[selmon->sellt]->arrange && !selmon->sel->isfloating)
+		return;
+	if (arg->i > 0) {
+		XRaiseWindow(dpy, selmon->sel->win);
+	} else {
+		if (!selmon->lt[selmon->sellt]->arrange)
+			XLowerWindow(dpy, selmon->sel->win);
+		else {
+			XWindowChanges wc;
+			wc.stack_mode = Above;
+			wc.sibling = barwin;
+			XConfigureWindow(dpy, selmon->sel->win, CWSibling | CWStackMode, &wc);
+		}
+	}
+}
+
